src/publish/common/publish.ts

```diff
@@ -33,6 +33,7 @@ import { formatResourcePath } from "../../core/resources.ts";
 import { encodeAttributeValue } from "../../core/html.ts";
 import { haveArrowKeys } from "../../core/platform.ts";
 import { capitalizeWord } from "../../core/text.ts";
+import { RenderFlags } from "../../command/render/types.ts";
```

Comment:
@dragonstyle Can you please review this file?

Reply:
done lgtm